core/domain/user_jobs_continuous_test.py

```diff
@@ -595,14 +596,17 @@ class UserImpactAggregatorTest(test_utils.GenericTestBase):
     EXPONENT = 2.0/3
 
     def setUp(self):
-        super(UserImpactAggregatorTest, self).setUp()
+        super(UserStatsAggregatorTest, self).setUp()
         self.num_completions = defaultdict(int)
+        self.num_starts = defaultdict(int)
 
     def _mock_get_statistics(self, exp_id, unused_version):
         current_completions = {
             self.EXP_ID_1: {
                 'complete_exploration_count': (
                     self.num_completions[self.EXP_ID_1]),
+                'start_exploration_count': (
```

Comment:
This is accessed inside user_jobs_continuous: https://github.com/526avijitgupta/oppia/blob/ad7d7fe78cb809a9c5afa135fa1d786b50e3a1d8/core/domain/user_jobs_continuous.py#L388

So, on removing this it throws me keyError because then the mocked data does not contain `start_exploration_count` key!